synapse/lib/view.py

```diff
@@ -340,6 +351,106 @@ def pack(self):
 
         await self.core.delView(self.iden)
 
+    def _allowed(self, user, parentlayr, perms):
```

Comment:
trying to continue the convention of these allowed methods that throw be called `_reqAllowed`